### PR TITLE
mediarithmics amp-analytics tag integration

### PR DESCRIPTION
**Pull request by PoulpiFr**
- Add mediarithmics integration in vendor.js of amp-analytics
- Add mediarithmics in amp-analytics examples page

Fixes/Closes/Related-to #11262

### CODE REVIEW

**Comment by PoulpiFr** [comment]
I signed it!

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -922,6 +922,34 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
...
+        '&$referrer=${referrer}',
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'pageview',
+      },
+    },
+    'transport': {
+      'beacon': false,
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js]: To confirm with you that the server does not support beacon xhrpost? They are recommended.

Comment (PoulpiFr) [reply extensions/amp-analytics/0.1/vendors.js]: Yes, for historical reasons, we are only supporting image/pixel mode today.

**Comment by PoulpiFr** [comment]
Hello,
I can't access percy interface and the Google CLA is signed but its status is not yet propagated on Github.

What can I do to move forward?

**Comment by PoulpiFr** [comment]
I was trolled by Googlebot...

**Comment by zhouyx** [comment]
@PoulpiFr. Sorry about the delay from CLA check. Yes the googlebot makes me confused and frustrated 😢  all the time.
Regarding the percy visual diff test. We've already skipped the flaky test here. Please rebase to latest master and it should fix the issue here. Thanks!

**Comment by PoulpiFr** [comment]
@zhouyx > Thanks for the feeback!

I just rebased to the lastest master!

**Pull request merged**